src/components/TreesMap/Tooltip.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { DataTable, TableItemsType } from '../DataTable';
+
+export const TOOLTIP_WIDTH = 260;
+
+const StyledTooltipWrapper = styled.div`
+  width: ${TOOLTIP_WIDTH}px;
+  position: absolute;
+  z-index: 1;
+  pointer-events: none;
+  box-shadow: ${({ theme }) => theme.boxShadow};
+  transform: translateY(10px);
+`;
+
+export const Tooltip: FC<{
+  x: number;
+  y: number;
+  title: string;
+  subtitle: string;
+  infos: TableItemsType;
+}> = ({ x, y, title, subtitle, infos }) => {
+  return (
+    <StyledTooltipWrapper
+      style={{
+        left: x - TOOLTIP_WIDTH / 2,
```

Comment:
This can be achieved without the need of hardcoding the tooltip's width by leveraging the CSS `transform` property. For example:

```js
<StyledTooltipWrapper
  style={{
    left: x,
    top: y,
  }}
  className='tooltip transform -translate-x-1/2 max-w-sm'
>
```

This is just a suggestion. Not a requirement. Feel free to ignore. :)

Reply:
True, good idea. Just remembered that also CSS `transform`s are much cheaper to render in comparison with a `left`.